pygbm/binning.py

```diff
@@ -124,11 +104,18 @@ def _map_num_col_to_bins(data, binning_thresholds, binned):
 
 
 class BinMapper(BaseEstimator, TransformerMixin):
-    """Transformer that maps a dataset into integer-valued bins
+    """Transformer that maps a dataset into integer-valued bins.
 
     The bins are created in a feature-wise fashion, with equally-spaced
     quantiles.
 
+    Large datasets are subsampled, but the feature-wise quantiles should
+    remain stable.
+
+    If the number of unique values for a given feature is less than
+    ``max_bins``, then the unique values of this feature are used instead of
```

Comment:
To be more accurate I used *...will be used to compute the bins thresholds...*

because the thresholds are the midpoints, not the exact unique values / quantiles.